modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/common/RESTTestBase.java

```diff
@@ -380,6 +380,18 @@ protected Response getResponseOfPost(String endpointUri, String body) {
                 .post(endpointUri);
     }
 
+    protected Response getResponseOfPostNoFilter(String endpointUri, String body) {
```

Comment:
Add a method comment

Reply:
This is updated with e24b571fe77c0a70265d04002039fee854912c9e